subsys/net/l2/bluetooth/bluetooth.c

```diff
@@ -379,7 +427,7 @@ static bool eir_found(u8_t type, const u8_t *data, u8_t data_len,
 
 		/* TODO: Notify device address found */
 		net_mgmt_event_notify(NET_EVENT_BT_SCAN_RESULT,
-				      bt_context_data.iface);
+				      bt_context_data.conns[0].iface);
```

Comment:
Is this index 0 correct here?

Reply:
Well it is only one that garanteed to exist afaik, though perhaps we tracking of which interface the scan command come from but at least the current shell will use the first interface it finds so at least it is correct in that regard.